requirements.txt

```diff
@@ -1,4 +1,4 @@
-# python_version 3.7.6
+# python_version 3.12
 #
```

Comment:
```suggestion
# pip_version 24.1.2
#
```

The jenkins failure is because we default to pip 20.x which doesn't seem to work with Python 3.12. Updating to the latest version of pip should help here.

Reply:
(this should be picked up automatically by JSL 0.32, unless it is specifically set